evalml/tests/component_tests/test_imputer.py

```diff
@@ -19,24 +19,24 @@ def imputer_test_data():
     return pd.DataFrame(
         {
             "categorical col": pd.Series(
-                ["zero", "one", "two", "zero", "three"], dtype="category"
+                ["zero", "one", "two", "zero", "two"] * 4, dtype="category"
```

Comment:
In this case it was easier to just lengthen the test data and allow woodwork to properly infer it as categorical wherever applicable.